mypy/fastparse.py

```diff
@@ -582,12 +582,14 @@ def visit_Set(self, n: ast35.Set) -> Node:
     # ListComp(expr elt, comprehension* generators)
     @with_line
     def visit_ListComp(self, n: ast35.ListComp) -> Node:
-        return ListComprehension(self.visit_GeneratorExp(cast(ast35.GeneratorExp, n)))
+        g = self.visit_GeneratorExp(cast(ast35.GeneratorExp, n))
+        return ListComprehension(cast(GeneratorExpr, g))
```

Comment:
Because visit_GeneratorExpr returns a Node bit ListComprehension takes a subclass thereof? Maybe the call was otherwise untyped?